src/content-handlers/iiif/modules/uv-mediaelementcenterpanel-module/MediaElementCenterPanel.ts

```diff
@@ -105,9 +105,15 @@ export class MediaElementCenterPanel extends CenterPanel<
       }
     });
 
-    this.extensionHost.subscribe(IIIFEvents.SET_MUTED, (muted) => {
+    this.extensionHost.subscribe(IIIFEvents.SET_MUTED, (muted: boolean) => {
       if (that.player) {
         that.player.setMuted(muted);
+
+        if (muted) {
+          that.$media.attr("muted", "");
+        } else {
+          that.$media.removeAttr("muted");
+        }
```

Comment:
This will add or remove the muted attribute when the mute is applied from outside of MediaElement, but it will not have the same effect if the volume is changed from inside MediaElement. For example, if I click the external mute control, the audio element gets a "muted" attribute. But if I then raise the volume in the player itself, the attribute remains. Do we need to account for this in the volumechange event handlers for consistency?